core/lib/runner.js

```diff
@@ -401,7 +401,12 @@ function runScenario(script, intermediate, runState, contextVars) {
       script.scenarios[i].weight);
 
   if(script.scenarios[i].apiVersions){
-    contextVars.contextVars = script.scenarios[i].apiVersions;
+    contextVars ?
```

Comment:
by looking at the code before, it looks like `contextVars` is defined, so i think you can just do

`contextVars.apiVersions = script.scenarios[i].apiVersions;` and that's it